R/errors.R

```diff
@@ -167,7 +167,7 @@ err <- local({
   #' @param ... Parts of the error message, they will be converted to
   #'   character and then concatenated, like in [stop()].
   #' @param call. A call object to include in the condition, or `TRUE`
-  #'   or `NULL`, meaning that [throw()] should add a call object
+  #'   or `NULL`, meaning that `throw()` should add a call object
```

Comment:
Why rewrite all these?

Reply:
Otherwise, recent roxygen2 complains. Try for yourself on the main branch.

Reply:
```
❯ packageVersion("roxygen2")
[1] ‘7.3.2’

❯ devtools::document()
ℹ Updating asciicast documentation
ℹ Loading asciicast
ℹ Re-compiling asciicast (debug build)
── R CMD INSTALL ───────────────────────────────────────────────────────────────
[...]
─  DONE (asciicast)
```